interface/modules/zend_modules/module/Patientvalidation/view/patientvalidation/patientvalidation/index.phtml

```diff
@@ -0,0 +1,118 @@
+<script>
+
+    /* +-----------------------------------------------------------------------------+
+     * Copyright 2016 matrix israel
+     * LICENSE: This program is free software; you can redistribute it and/or
+     * modify it under the terms of the GNU General Public License
+     * as published by the Free Software Foundation; either version 3
+     * of the License, or (at your option) any later version.
+     * This program is distributed in the hope that it will be useful,
+     * but WITHOUT ANY WARRANTY; without even the implied warranty of
+     * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+     * GNU General Public License for more details.
+     * You should have received a copy of the GNU General Public License
+     * along with this program. If not, see
+     * http://www.gnu.org/licenses/licenses.html#GPL
+     *    @author  Dror Golan <drorgo@matrix.co.il>
+     * +------------------------------------------------------------------------------+
+     *
+     */
+    <?php if ($closeBeforeOpening):?>
+    opener.document.forms[0].submit();
+    window.close();
+
+    <?php endif;?>
+</script>
+<table id="example" class="display">
+    <thead>
+    <th><?php echo $translate->z_xlt("ID");?></th>
+    <th><?php echo $translate->z_xlt("First name");?></th>
+    <th><?php echo $translate->z_xlt("Last name");?></th>
+    <th><?php echo $translate->z_xlt("DOB");?></th>
+    <th><?php echo $translate->z_xlt("Sex");?></th>
+    </thead>
+    <tbody>
+
+    <?php if ($related_patients) : ?>
+        <?php foreach ($related_patients as $key=>$related_patient):?>
+            <tr>
+                <td><?php echo( $translate->z_xlt($related_patient['id']));?></td>
+
+                <td><?php echo $translate->z_xlt($related_patient['fname']);?></td>
+                <td><?php echo $translate->z_xlt($related_patient['lname']);?></td>
+                <td><?php echo $translate->z_xlt($related_patient['DOB']);?></td>
```

Comment:
yep, Zend uses that function to escape html. Just to provide an example, if the id was populated by:
``````
<script>alert("hi there, I am going to somehow steal your session via javascript")</script>
``````
the escapeHtml will remove the above vulnerability by escaping the <,> chars